test/parameter_test.py

```diff
@@ -261,6 +275,16 @@ def test_enum_param_invalid(self):
     def test_enum_param_missing(self):
         self.assertRaises(ParameterException, lambda: luigi.parameter.EnumParameter())
 
+    def test_list_parameter(self):
+        a = luigi.ListParameter()
+        b_list = [1, 2, 3]
+        self.assertEqual(FooList.a_list, a.parse(a.serialize(b_list)))
```

Comment:
I think you are trying to test that when you take a python value and ser-de it it ends up at the same python value, no? So you want something like 

`assertEqual(b_list, a.parse(a.serialize(b_list))`